arch/arm/src/stm32f7/stm32_serial.c

```diff
@@ -3641,10 +3641,10 @@ void arm_serialinit(void)
 
 #if !defined(SERIAL_HAVE_ONLY_DMA)
 #  if defined(SERIAL_HAVE_RXDMA)
-  UNUSED(g_uart_rxdma_ops);
+  UNUSED(&g_uart_rxdma_ops);
```

Comment:
@davids5 

Is this change intended to be included? It stands out from everything else.

Reply:
Yes. it is listed "stm32f7:serial fix compile error from UNUSED() change" 
UNUSED changed recently and broke it.

Reply:
Cool thanks for the explanation.